### ISSUE TITLE
Add security for admin console to review current registered users.

### ISSUE THREAD

**Issue by varunmehta** [varunmehta/gric-diwali#8]

**Comment by kunalkhosla** [comment]
Would this also mean registration page for the user? Do users create logins?

**Comment by kunalkhosla** [comment]
Self note : admins will be able to create events. The POST for event creation will be authorized by spring security

**Comment by varunmehta** [comment]
> Would this also mean registration page for the user? Do users create logins? 

No only Admins, users will come and go per event. They have no logins for now.

**Comment by kunalkhosla** [comment]
So im guessing there is an "Admin Login" button somewhere, which admins will use to login. To its success, they can manage events. Let me know if im misunderstanding.

**Comment by varunmehta** [comment]
Nope, you got it perfectly correct, for now we can secure the REST call, and then make a simple page to handle it. Do you know any good ui tech which is quick & easy to use (unlike angular) and can be used to make this UI ?

**Comment by kunalkhosla** [comment]
Simple http and jquery. Rest all are learning curve for me
